packages/toolchain/elemental-cli/collection.yaml

```diff
@@ -3,14 +3,14 @@ packages:
     name: "elemental-cli"
     category: "toolchain"
     bin_name: "elemental"
-    version: 0.0.13-0
+    version: "0.20220124"
     fips: false
     labels:
       github.repo: "elemental-cli"
       github.owner: "rancher-sandbox"
       autobump.revdeps: "true"
       autobump.strategy: "git_hash"
-      git.hash: "d1d11006f556aacaddf82a403894a4445b7b696c"
+      git.hash: "null"
```

Comment:
come on

Reply:
ah of course the autobumper uses plain curl, I thought it used the -L flag to follow up links...